build.gradle.kts

```diff
@@ -101,8 +101,13 @@ tasks {
       .singleFile
     jvmArgs = listOf("-javaagent:$jacocoAgent=destfile=$buildDir/jacoco/cucumber.exec,append=false")
   }
+  val versionTwo by registering(Test::class) {
```

Comment:
This is getting messy, lets sort this out Thursday.

Reply:
It's a nightmare. We also need to consider the BDD tests which are an accident waiting to happen. We have got three versions of the application now and the tests cannot keep up as they are currently structured